.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false # Use new Travis container-based infrastructure
```

Comment:
Does this mean you don't intend to handle using Circle in this PR? It is indeed very simple to tell each provider which branches to build (would it be only by ticking “build only if `travis|circle.yml` is present”). I can help if you want.